ert/data/record/_transformation.py

```diff
@@ -59,6 +70,20 @@ async def transform_input(
             tar.extractall(runpath / location)
 
     async def transform_output(self, mime: str, location: Path) -> Record:
+        def sync_make_tar(file_path: Path) -> bytes:
```

Comment:
Not a big fan of these local functions, but maybe it's allright in this ace

Reply:
And test potentially, but I had a suggestion further down for it.

Reply:
Yes, as long as they remain private functions that would be possible, and maybe clearer. I will see how that looks.